tests/rafter/README.md

```diff
@@ -70,7 +70,7 @@ Use the following environment variables to configure the application:
 | **APP_TEST_COMMON_ASSET_PREFIX**      | No       | `test`                     | The name of the prefix for the Asset and ClusterAsset resources                                                                             |
 | **APP_TEST_MOCKICE_NAME**             | No       | `rafter-test-svc`          | The name of the pod, service, and configmap used by the test service                                                                        |
 
-Those can be supplied to [this](../../resources/rafter/templates/tests/test.yaml) file before installing Kyma, or by editing TestDefinition CR with already installed Kyma using this command:
+Those can be supplied to this file before installing Kyma, or by editing TestDefinition CR with already installed Kyma using this command:
```

Comment:
Instead of only removing the link, shouldn't the whole section be removed? 
Line 73-77
"Those can be supplied to this file before installing Kyma, or by editing TestDefinition CR with already installed Kyma using this command:
k edit testdefinitions.testing.kyma-project.io -n kyma-system rafter"